provisioner/powershell/provisioner_test.go

```diff
@@ -381,6 +381,39 @@ func TestProvisionerProvision_ValidExitCodes(t *testing.T) {
 	}
 }
 
+func TestProvisionerProvision_PauseAfter(t *testing.T) {
+	config := testConfig()
+	delete(config, "inline")
+
+	// Defaults provided by Packer
+	config["remote_path"] = "c:/Windows/Temp/inlineScript.ps1"
+	config["inline"] = []string{"whoami"}
+	ui := testUi()
+	p := new(Provisioner)
+
+	// Defaults provided by Packer
+	p.config.PackerBuildName = "vmware"
+	p.config.PackerBuilderType = "iso"
+	p.config.ValidExitCodes = []int{0, 200}
+	pause_amount := time.Duration(1 * time.Second)
+	p.config.PauseAfter = pause_amount
+	comm := new(packersdk.MockCommunicator)
+	comm.StartExitStatus = 200
+	p.Prepare(config)
```

Comment:
If you run `make ci-lint` locally you'll see the linter yelling at this statement since `p.Prepare` returns an error, which is not checked.
For the sake of consistency with the rest of the file I'd assume you can leave it this way, but if you want to address it, feel free to.

Reply:
Updated to checked for error